app/dao/users_dao.py

```diff
@@ -13,15 +14,17 @@
 from app.models import EMAIL_AUTH_TYPE, User, VerifyCode
 from app.utils import escape_special_characters, get_archived_db_column_value
 
+MFA_TOTP_DEFAULT_LENGTH = int(os.getenv('MFA_TOTP_LENGTH', 6))
```

Comment:
This variable name is possibly confusing, as 6 is the default unless changed via an env var. I'm not immediately thinking of a better naming scheme, so this is probably fine unless something pops out to you.

Reply:
Oh good point!  I'll rename that so it's less confusing.